dogowner/test_dog_owner.py

```diff
@@ -178,3 +178,21 @@ def test_dog_owner_user_creation_with_non_alfa_dog_name(self):
                             phone_number=PHONE_NUMBER, dog_race=DOG_RACE, dog_picture_url=DOG_PICTURE_URL,
                             dog_age=DOG_AGE, dog_weight=DOG_WEIGHT, dog_gender=DOG_GENDER
                             )
+
+    def test_dog_owner_user_creation_with_invalid_url(self):
```

Comment:
I'm pretty sure Django has a URLField
might be better to change to this instead of CharField, therefore making the test redundant. 
I also think you can add a validator to it, using the `validate_url_is_image` func

Reply:
Sounds good .

Reply:
After changing form CharField to UrlField , django create 0003_alter_dogowner_dog_picture_url.py file .
Do you think I should squash 0003 with 0001 ? This question was raise because of https://github.com/redhat-beyond/Hotails/pull/65#discussion_r847086629

Reply:
You should never modify migration files by yourself; let django auto migrations do its work